nixos/modules/services/monitoring/glpi-agent.nix

```diff
@@ -0,0 +1,94 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.glpiAgent;
+
+  settingsType =
+    with lib.types;
+    attrsOf (oneOf [
+      bool
+      int
+      str
+      (listOf str)
+    ]);
+
+  formatValue =
+    v:
+    if lib.isBool v then
+      if v then "1" else "0"
+    else if lib.isList v then
+      lib.concatStringsSep "," v
+    else
+      toString v;
+
+  configContent = lib.concatStringsSep "\n" (
+    [
+      "# Server configuration"
```

Comment:
Why is this special cased? Does it need to go first?

Reply:
It's not mandatory.
But for me and my team it's more readable to see the server where it is quickly it's simpler and it's not a very big modification.

Reply:
So I want to keep this if possible.

Reply:
I don't think it's worth it, given you can just `cat $configfile | grep server`, or eval it directly from the NixOS config.

Reply:
I know all that.
But I don't see the problem with what I added.
It makes the file more readable.
Do you have a policy that states that any generated file should not have a specific treatment for better readability.
Please know that I am open to discussion on the subject and to the good practices you have on nixos.
But it is also important to me to make this accessible to as many people as possible.

Reply:
I think the readability of the NixOS module is more important than the readability of the generated file, and when I see this kind of construct, it makes me question things like "does the order of entries actually matter".